docs/python/testing.md

```diff
@@ -306,30 +162,41 @@ For example, the configuration below in the `launch.json` file disables the `jus
-|  **Test: Debug Test at Cursor** | Debug the test method where you have your cursor focused on the editor. Similar to **Python: Debug Test Method...** on versions prior to 2021.9. |
-|  **Test: Debug Tests in Current File** | Debug tests in the file that is currently in focus on the editor. |
-|  **Test: Go to Next Test Failure** | If the error peek view is open, open and move to the peek view of the next test in the explorer that has failed. |
-|  **Test: Go to Previous Test Failure** | If the error peek view is open, open and move to the peek view of the previous test in the explorer that has failed. |
-|  **Test: Peek Output** | Opens the error peek view for a test method that has failed. |
-|  **Test: Refresh Tests** | Perform test discovery and updates the Test Explorer to reflect any test changes, addition, or deletion. Similar to **Python: Discover Tests** on versions prior to 2021.9. |
-|  **Test: Rerun Failed Tests** | Run tests that failed in the most recent test run. Similar to **Python: Run Failed Tests** on versions prior to 2021.9.  |
-|  **Test: Rerun Last Run** | Debug tests that were executed in the most recent test run.  |
-|  **Test: Run All Tests** |  Run all discovered tests. Equivalent to **Python: Run All Tests** on versions prior to 2021.9.  |
-|  **Test: Run Test at Cursor** |  Run the test method where you have your cursor focused on the editor.  Similar to **Python: Run Test Method...** on versions prior to 2021.9. |
-|  **Test: Run Test in Current File** | Run tests in the file that is currently in focus on the editor. Equivalent to **Python: Run Current Test File** on versions prior to 2021.9. |
-|  **Test: Show Output** | Open the output with details of all the test runs. Similar to **Python: Show Test Output** on versions prior to 2021.9. |
-|  **Testing: Focus on Test Explorer View** | Open the Test Explorer view. Similar to **Testing: Focus on Python View** on versions prior to 2021.9.
-|  **Test: Stop Refreshing Tests** | Cancel test discovery. |
+1. Open the integrated terminal and install the `pytest-xdist` package. For more details, refer to the [project's documentation page](https://pypi.org/project/pytest-xdist/).
+
+   **For Windows**
+
+   ```bat
+   py -3 -m pip install pytest-xdist
+   ```
+
+   **For macOS/Linux**
+
+   ```bash
+   python3 -m pip install pytest-xdist
+   ```
+
+2. Next, create a file named `pytest.ini` in your project directory and add the content below, specifying the number of CPUs to be used. For example, to set it up for 4 CPUs:
+
+   ```ini
+    [pytest]
+    addopts=-n4
+   ```
+
+   Or, if you are using a `pyproject.toml` file
+
+   ```toml
+    [tool.pytest.ini_options]
+    addopts="-n 4"
+   ```
+
+If `xdist` is enabled and no worker count is specified in the arguments, the number of workers is automatically optimized by the Python extension based on how many tests were selected in the Test Explorer.
```

Comment:
I feel like explaining how to install xdist and where to put it in the init file is too verbose / not our job. I would maybe shrink this to just discuss that it can be configured. ALso you can always add -n=4 to the pytest args which is easier than these config files too